<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterMinMaxView</class>
 <widget class="QWidget" name="FilterMinMaxView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="filterGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbFilter">
        <property name="text">
         <string>Enable filtering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="filterMinLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Filter Minimum</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="filterMinSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>450</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="filterMaxLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Filter Maximum</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="filterMaxSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>450</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>450</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterMaxLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterMaxSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterMinLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterMinSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
